rskj-core/src/test/java/co/rsk/peg/BridgeSupportAddSignatureTest.java

```diff
@@ -57,11 +56,14 @@
 
 class BridgeSupportAddSignatureTest {
     private static final RskAddress bridgeAddress = PrecompiledContracts.BRIDGE_ADDR;
-    private static final List<BtcECKey> federationECKeys = BitcoinTestUtils.getBtcEcKeysFromSeeds(
-        new String[]{"fa91", "fa92", "fa93"}, true
+    private static final List<BtcECKey> REGTEST_FEDERATION_PRIVATE_KEYS = Arrays.asList(
+        BtcECKey.fromPrivate(Hex.decode("45c5b07fc1a6f58892615b7c31dca6c96db58c4bbc538a6b8a22999aaa860c32")),
```

Comment:
You should get these private keys from the `FederationTestUtils`

Reply:
Yep, and found some others tests where I didn't delete the constant